src/EnergyPlus/SystemReports.cc

```diff
@@ -3915,6 +3915,7 @@ void CalcSystemEnergyUse(EnergyPlusData &state,
         COILSYSTEM_COOLING_DX,
         COILSYSTEM_COOLING_DX_HEATEXCHANGERASSISTED,
         COILSYSTEM_COOLING_WATER_HEATEXCHANGERASSISTED,
+        COILSYSTEM_COOLING_WATER,
```

Comment:
Adds  enum type "COILSYSTEM_COOLING_WATER" for use with CalcSystemEnergyUse() function and system reports.